src/ui/version.coffee

```diff
@@ -15,7 +15,7 @@ check = (noAlert = false)->
         return console.log err if err
         body = JSON.parse body
         tag = body.tag_name
-        releasedVersion = tag.substr(1) # remove first "v" char
```

Comment:
the about window is not implemented properly in the trifl framework as it is a static window.

I didn't know better at the time and didn't implement separated from the app (in the sense that as soon as the app was being drawn the first time, it also was drawing the about.. in turn was requiring version check that would fail.

By some reason the request would not stop in the `if error` condition and would throw an exception at this point. As there is a `releasedVersion?` check next that would catch this gracefully.

Anyway, I need to talk to you about how best to implement a static window in trifl and probably refactor the about window